integration_testing/features/going-back-from-review-selections-for-import-returns-to-channel-list.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Going back from Review selections for import returns to channel list
+
+	Background: 
+		Given that I am already a registered user
+			And I am logged-in to the system
+			And I am on *My Channels > edit* page

```

Comment:
This can all be condensed into one line, first 2 steps are unnecessary: if a user is on the `*My Channels > '<channel>'` page, they are already registered and signed in, right? :wink: